src/ring_theory/fractional_ideal.lean

```diff
@@ -139,10 +143,11 @@ lemma mem_zero_iff {x : P} : x ∈ (0 : fractional_ideal f) ↔ x = 0 :=
     by simp [x'_eq_x.symm, x'_eq_zero]),
   (λ hx, ⟨0, rfl, by simp [hx]⟩) ⟩
 
-@[simp] lemma val_zero : (0 : fractional_ideal f).1 = 0 :=
+@[simp] lemma coe_zero : ↑(0 : fractional_ideal f) = (⊥ : submodule R f.codomain) :=
 submodule.ext $ λ _, mem_zero_iff
 
-lemma nonzero_iff_val_nonzero {I : fractional_ideal f} : I.1 ≠ 0 ↔ I ≠ 0 :=
+lemma nonzero_iff_coe_nonzero {I : fractional_ideal f} :
```

Comment:
Should this be `coe_ne_bot_iff_nonzero`?

Reply:
It should, thanks!